Objects/longobject.c

```diff
@@ -274,44 +274,40 @@ _PyLong_Negate(PyLongObject **x_p)
 }
 
 /* Create a new int object from a C long int */
+
```

Comment:
This is a minor consistency fix, following the [Boy Scout Rule](https://wiki.c2.com/?BoyScoutRule): the prevailing style in this file is to leave a blank line between the comment-description and the definition. But the change is not strictly necessary for this PR, and I can revert if reviewers prefer.